enh: replace unix2dos with sed (#3210)
uxlfoundation/oneDAL: INSTALL.md

* oneDPL library
* Microsoft Visual Studio\* (Windows\* only)
* [MSYS2](http://msys2.github.io) (Windows\* only)

> * `make`tool; install these packages using MSYS2 on Windows\* as follows:

Reviewer: ```suggestion
* `make`; which can be installed using MSYS2 on Windows\* as follows:
```
